test/libsolidity/syntaxTests/pragma/experimental_solidity_out_of_order_1.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.8.19;
```

Comment:
Don't add version pragmas to tests - what's the purpose of this? It'll be in the preamble already anyways. Same below.

Reply:
Yup, now that you mention it, the last test `out_of_order_3` is redundant, but then I only learned about the preamble last week. The issue here is that I want the pragma (in this case version) to be explicit, because that's exactly the behaviour I'm testing - the order of pragmas; I can certainly remove these, but the prerequisite for anyone looking at the test would be to know about the implicit preamble? Your decision.

Reply:
Remove it :-).